CMakeLists.txt

```diff
@@ -157,12 +157,6 @@ if(UNIX)
             "System-wide search directory. If not set or empty, CMAKE_INSTALL_FULL_SYSCONFDIR and /etc are used.")
 endif()
 
-# Because we use CMake 3.10.2, we can't use the policy which would disable adding /W3 by default. In the interim, replace the flags
-# When this project is updated to 3.15 and above, use the following line.
-#     cmake_policy(SET CMP0092 NEW)
-string(REGEX REPLACE "/W3" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
-string(REGEX REPLACE "/W3" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
By updating to 3.17, the warning should disappear on its own?

Reply:
Correct updating the minimum also updates the policy in question

Reply:
This actually made me realize a regression in vvl: https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/5182